deployment/compose.env.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
This script can be used to source the environment variable when you need to execute docker-compose directly.
To use:

```
source compose.env.sh
```